### DISCUSSION TITLE
Use Hurl logo in article for my blog

### DISCUSSION THREAD

**Discussion by rockchalkwushock** [Orange-OpenSource/hurl#2501] [General] [answered]
Hey! Came across Hurl recently and I'm using it within a Go project. I am writing an article about using Hurl with Go and was looking for your logo on https://svgporn.com/ to use in the hero image for the article but could not find it there. I just wanted to ask if it's okay to use the logos found here in the repository?

**Comment by jcamiel** [comment, marked as answer]
Yes of course!

**Reply by rockchalkwushock** [reply]
Awesome thanks for the quick response @jcamiel

**Reply by jcamiel** [reply]
You're welcome; share the link to your blog post when it's written, please!

**Reply by rockchalkwushock** [reply]
Absolutely will do, thanks to you and all the maintainers and contributors Hurl is awesome!
